client/src/commands/run.ts

```diff
@@ -29,13 +29,31 @@ interface FoldingBlock {
 let running = false;
 
 function getCode(selected = false, uri?: Uri): string {
+  if (uri) {
+    console.log("run.ts::getCode() - uri.fsPath:", uri.fsPath && uri.fsPath);
+  }
   const editor = uri
     ? window.visibleTextEditors.find(
         (editor) => editor.document.uri.toString() === uri.toString(),
       )
     : window.activeTextEditor;
   const doc = editor?.document;
   let code = "";
+  let setCodeFile = "";
+  if (uri && uri.fsPath) {
+    setCodeFile =
+      "option set=SAS_EXECFILEPATH=%sysfunc(quote(" + uri.fsPath + "));\n";
```

Comment:
it would be great to extract the statement to a single place to be used in multiple places.

Reply:
will do